### ISSUE TITLE
Sub-folder Support still requires TENANT_DOMAIN_MODEL to be set.

### ISSUE THREAD

**Issue by vforvoid** [django-tenants/django-tenants#490]
Hi there and first of all thank you and all the contributors for this awesome project!

I was looking through sub-folder examples and I noticed that it still have a lot of code related to sub-domain setup, for example:

1. [In settings](https://github.com/django-tenants/django-tenants/blob/master/examples/tenant_subfolder_tutorial/tenant_subfolder_tutorial/settings.py#L173) in models
2. [In models](https://github.com/django-tenants/django-tenants/blob/master/examples/tenant_subfolder_tutorial/customers/models.py#L14)
3. [In admin](https://github.com/django-tenants/django-tenants/blob/master/examples/tenant_subfolder_tutorial/customers/admin.py#L6)

Is it somehow possible to make domains optional?

**Comment by vforvoid** [comment]
Yes, I was reading exactly the example you're talking about and all links in OP are leading there. And yes, I guess "domain" name should be generalized to something that covers both domains and subfolders.

**Comment by tomturner** [comment]
Correct
